test/AsmJs/rlexe.xml

```diff
@@ -2,6 +2,18 @@
 <regress-exe>
   <test>
     <default>
+      <files>argTest.js</files>
+      <compile-flags>-maic:0</compile-flags>
+    </default>
+  </test>
+  <test>
+    <default>
+      <files>argTestMixed.js</files>
```

Comment:
maybe but we don't have simd is enabled. These tests are now useful for xplat and will be useful for other archs.